packages/dropdowns/src/elements/Menu/Items/MediaFigure.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLAttributes } from 'react';
+import { StyledMediaFigure } from '../../../styled';
+import useMenuContext from '../../../utils/useMenuContext';
+
+/**
+ * Accepts all `<div>` props
+ */
+export const MediaFigure: React.FC<HTMLAttributes<any>> = props => {
```

Comment:
Should this component forward refs?

Reply:
Answered in https://github.com/zendeskgarden/react-components/pull/600#discussion_r372666448